src/AppBundle/Helper/AutomatedEditsHelper.php

```diff
@@ -65,13 +72,45 @@ public function isAutomated(string $summary, Project $project): bool
         return (bool)$this->getTool($summary, $project);
     }
 
+    /**
+     * Fetch the config from https://meta.wikimedia.org/wiki/MediaWiki:XTools-AutoEdits.json
+     * @param bool $useSandbox
```

Comment:
Please document that $useSandbox results in a different config source - the current documentation suggest that the main XTools-AutoEdits.json is always used